assets/text/effects/wildermyth-drauven-pcs_outpostCelebration.properties

```diff
@@ -0,0 +1,6 @@
+#suppress inspection "UnusedProperty" for whole file
```

Comment:
Minor nitpick: We can make an unlimited number of branches & merge requests, so a good practice is to keep 1 branch = 1 feature or comic.  Not a big deal for this one, but by splitting things up into smaller parts:
* It's generally easier to keep track of how everything is related during the code review, and
* If one feature sparks a bunch of discussion, splitting things up into small chunks means that the other pieces can be merged in while the interesting discussion parts are being resolved.